src/content/guides/asset-management.md

```diff
@@ -380,50 +351,20 @@ npm install --save-dev csv-loader xml-loader
 __webpack.config.js__
 
 ``` diff
-  const path = require('path');
-
-  module.exports = {
-    entry: './src/index.js',
-    output: {
-      filename: 'bundle.js',
-      path: path.resolve(__dirname, 'dist'),
-    },
-    module: {
-      rules: [
-        {
-          test: /\.css$/,
-          use: [
-            'style-loader',
-            'css-loader'
-          ],
-        },
-        {
-          test: /\.(png|svg|jpg|gif)$/,
-          use: [
-            'file-loader',
-          ],
-        },
-        {
-          test: /\.(woff|woff2|eot|ttf|otf)$/,
-          use: [
-            'file-loader',
-          ],
-        },
-+       {
-+         test: /\.(csv|tsv)$/,
-+         use: [
-+           'csv-loader',
-+         ],
-+       },
-+       {
-+         test: /\.xml$/,
-+         use: [
-+           'xml-loader',
-+         ],
-+       },
-      ],
-    },
-  };
+         test: /\.(woff|woff2|eot|ttf|otf)$/i,
```

Comment:
I think it is better to keep the entire diff, in this case you are replacing the entire content, so it is not longer a diff.